### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by DoubangoTelecom** [Nikit333/Face-Anti-Spoofing-Android-iOS#2]
Hello,
We have tested your implementation but it's not accurate on high definition images. According to issue #1 it uses Huawei's liveness detection (ML Kit). We built an open source implementation based on Huawei's code and it's also not accurate: https://github.com/DoubangoTelecom/HuaweiFaceLiveness

Are you planning to improve the accuracy?
Do you have Linux version we can test?

**Comment by Nikit333** [comment]
Now I completed training new model, and accuracy is good on my dataset.
If you want, I can give you web api access.
And I tested your web api, but in a few tests, I could spoof it with my db files.


![image](https://user-images.githubusercontent.com/80960135/115805737-244b0a00-a418-11eb-9b6f-ec7241f61e2c.png)

**Comment by DoubangoTelecom** [comment]
Please share sample images and we'll check what is wrong

**Comment by DoubangoTelecom** [comment]
Are you selling your dataset?

**Comment by Nikit333** [comment]
Well, it's inappropriate here, you can contact me on skype.

**Comment by DoubangoTelecom** [comment]
> Now I completed training new model, and accuracy is good on my dataset.
> If you want, I can give you web api access.
> And I tested your web api, but in a few tests, I could spoof it with my db files.
> 
> Strange, almost all my spoof files are determined by 'genuine'.
> Are there any bug there?
> 
> ![image](https://user-images.githubusercontent.com/80960135/115805737-244b0a00-a418-11eb-9b6f-ec7241f61e2c.png)
> 
> ![image](https://user-images.githubusercontent.com/80960135/115806322-206bb780-a419-11eb-8bf7-02b306c21ec7.png)
> 
> ![image](https://user-images.githubusercontent.com/80960135/115806432-56a93700-a419-11eb-8f1c-54388be8c4ca.png)
> 
> ![image](https://user-images.githubusercontent.com/80960135/115806695-d33c1580-a419-11eb-868a-cf6c75de0109.png)

Yes, it's normal that your spoof images are accepted. They don't match the requirements but returned a result to ease testing. we'll change the code to reject images not matching these requirements. More info at https://github.com/DoubangoTelecom/FaceLivenessDetection-SDK/issues/3. 
Thanks for your feedback, it helped us improve the accuracy.
